frontend/tests/playwright/fixtures/product-page.ts

```diff
@@ -89,4 +89,19 @@ export class ProductPage {
    async switchSelectedVariant(): Promise<void> {
       await (await this.getInactiveVariant()).click();
    }
+
+   /**
+    * @description Will assert that the inventory message has the given message. If no message is given, it will assert that the inventory message is not visible.
```

Comment:
[minor] This is a pretty long line. Can we split it onto two lines

Reply:
Addressed in [3b1627b](https://github.com/iFixit/react-commerce/pull/1268/commits/3b1627b0bdbb173b151c7bbe16838f6f0824f06a)